vllm/executor/msgspec_utils.py

```diff
@@ -0,0 +1,26 @@
+from array import array
+from typing import Any, Type
+
+
+def encode_hook(obj: Any) -> Any:
+    """Custom msgspec enc hook that supports array types.
+
+    See https://jcristharif.com/msgspec/api.html#msgspec.msgpack.Encoder
+    """
+    if isinstance(obj, array):
+        return obj.tobytes()
```

Comment:
do you need to make sure the type is `I` here?

Reply:
good point! added